mob/src/main/java/com/github/phantazmnetwork/mob/spawner/BasicMobSpawner.java

```diff
@@ -83,16 +188,19 @@ private void setHealth(@NotNull NeuralEntity entity, @NotNull MobModel model) {
     }
 
 
-    private void addGoals(@NotNull PhantazmMob mob) {
-        for (Collection<Goal> group : mob.model().getGoalGroups()) {
-            Collection<NeuralGoal> neuralGroup = new ArrayList<>(group.size());
-            for (Goal goal : group) {
-                neuralGroup.add(goal.createGoal(mob));
-            }
+    private void addGoals(@NotNull ElementContext context, @NotNull DependencyProvider dependencyProvider,
+            @NotNull NeuralEntity entity, @NotNull MobModel model) {
+        Collection<GoalGroup> goalGroups =
+                true ? Collections.emptyList() : context.provide("goalGroups", dependencyProvider);
```

Comment:
goalGroups is always going to be Collection.emptyList() (is this left over from a refactor? XD)